### ISSUE TITLE
crashes with last.fm if no album art

### ISSUE THREAD

**Issue by surasshu** [jac0b-w/AlbumPaper#22]
to replicate: use the last.fm API functionality, then play any song that has no album art on last.fm.

i love this program, it's very nice to have the art change with every song! i hope this can be fixed because i would really like to keep using it, but i don't have spotify.

**Comment by jac0b-w** [comment]
Hey a few things:
1. What version of AlbumPaper are you using?
2. Could you give an example of a song (or multiple)
3. If you could send a copy of the `errors.log` file which is located in the installation folder (default is C:\AlbumPaper)

Also correct me if I'm wrong but you are playing local files and have a last.fm scrobbler installed.

Glad you like the program and hopefully I can get this fixed soon :)

**Comment by surasshu** [comment]
1. this is about version 3.3--i did try the 4.0 beta but it had a different error (i made a separate report of that one)
2. yes! i was wrong about not having album art, upon further testing--i think it's actually due to a song not having anything for the album tag at all that's causing the error. I've attached a song that causes the crash on my end
[Tancla - Ceramic Ivy.zip](https://github.com/jac0b-w/AlbumPaper/files/9500808/Tancla.-.Ceramic.Ivy.zip)
3. of course--see attached!
[errors.log](https://github.com/jac0b-w/AlbumPaper/files/9500894/errors.log)

and yes, i'm playing local MP3s with a last.fm scrobbler.

**Comment by jac0b-w** [comment]
Fantastic! I think this should be fixed in the next beta (hopefully soon) although since I can't easily replicate your exact setup I'll leave the issue open until I know for sure :)

**Comment by jac0b-w** [comment]
@surasshu Was this issue fixed in v4.0? 🤞

**Comment by surasshu** [comment]
yes--it no longer crashes from what i can tell! there's a very minor issue in that sometimes the wallpaper reverts back to the default wallpaper (usually the album art comes back a few seconds later) and i wanted to write a proper bug report for that, but i haven't been able to figure out what is causing it. it's probably not related so we can probably close this!

**Comment by jac0b-w** [comment]
Usually that's caused by a poor connection or it's possible it's due to the polling of your scobbler to last.fm. Ultimately unless my code is doing something that's not aligning with the playing status in the last.fm API response it's unlikely to be something I can fix although feel free to create an issue anyway.

Anyways I'm going to close this issue :)
